### PR TITLE
Updating editors manual docs for Wagtail 1.0

### PR DESCRIPTION
**Pull request by chrxr**
* Removed all references to Wagtail demo
* Now references Torchbox.com as example site, as it includes all new features
* Changed many screenshots

To do:

* [x] Snippets
* [ ] Any additional new features

May as well merge these changes in though.

### CODE REVIEW

File: docs/editor_manual/documents_images_snippets/images.rst [hunk 1]
```diff
@@ -5,9 +5,15 @@ If you want to edit, add or remove images from the CMS outside of the individual
 
 .. image:: ../../_static/images/screen31_images_page.png
 
-* Clicking an image will allow you to edit the data associated with it. This includes the Alt text, the photographers credit, the medium of the subject matter and much more. **NOTE:** changing the alt text here will alter it for all occurrences of the image in carousels, but not in inline images, where the alt text can be set separately.
+* Clicking an image will allow you to edit the data associated with it. This includes the Alt text, the photographers credit, the medium of the subject matter and much more. 
+
+.. Warning:: 
+	changing the alt text here will alter it for all occurrences of the image in carousels, but not in inline images, where the alt text can be set separately.
```
Comment (zerolab) [inline docs/editor_manual/documents_images_snippets/images.rst]: s/changing/Changing/

File: docs/editor_manual/new_pages/creating_body_content.rst [hunk 2]
```diff
@@ -1,13 +1,39 @@
 Creating page body content
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The majority of the body content within a page will be created within text fields.
+Wagtail supports a number of basic fields for creating content, as well as our unique Streamfield feature which allows you to construct complex layouts by combining these basic fields in any order.
 
-There are two types of text entry fields you will see when creating a page. Some, like *Page title* are basic text fields come with no formatting options. This is because all titles display the same across a single page type. Others, *Body* for example, require more formatting, and so a set of tools are provided for this purpose. These are referred to as rich text fields. 
+===========
+Streamfield
```
Comment (zerolab) [inline docs/editor_manual/new_pages/creating_body_content.rst]: I believe it should be "StreamField" everywhere, as per http://docs.wagtail.io/en/latest/topics/streamfield.html

**Comment by SalahAdDin** [comment]
But, i think so that wagtail demo is most easier and have one of thing that could be useful.

**Comment by chrxr** [comment]
Thanks @zerolab, I've made those fixes.
Also updated snippets section.

**Pull request merged**